test/resources/api-1999-bonds-only-valid-example.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Would it be useful to add these examples to postman? I'm not sure, just wanted to check your opinion @cyeniceri

Reply:
It would be useful for testers, so I added them (which we are using atm, and will be adding more once we add the functionality) and squash these 2 commits into 1